packages/commerce-sdk-react/src/hooks/ShopperBaskets/config.ts

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
A thought on this name file, naming this file config.ts is very vague as to what it contains. How about cache.ts or cache-update.ts?

Reply:
I originally gave it a generic name in case there's ever other manual config that we want to set. But now that I've seen how extensive the cache logic is, I think it makes sense to rename the file, and if we have a future need for extra config, put that somewhere else.